provision/minikube/keycloak/config/kcb-infinispan-cache-remote-store-config.xml

```diff
@@ -0,0 +1,261 @@
+                              cache="sessions"
+                              raw-values="true"
+                              shared="true"
+                              segmented="false">
+                    <remote-server host="${env.KC_REMOTE_STORE_HOST}"
+                                   port="${env.KC_REMOTE_STORE_PORT}"/>
+                    <connection-pool max-active="16"
+                                     exhausted-action="CREATE_NEW"/>
+                    <security>
+                        <authentication server-name="infinispan">
+                            <digest username="${env.KC_REMOTE_STORE_USERNAME}"
+                                    password="${env.KC_REMOTE_STORE_PASSWORD}"
+                                    realm="default"/>
+                        </authentication>
+                        <encryption protocol="TLSv1.3"
+                                    sni-hostname="${env.KC_REMOTE_STORE_HOST}">
+                            <truststore filename="/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+                                        type="pem"/>
+                        </encryption>
+                    </security>
+                </remote-store>
+            </persistence>
+            <state-transfer enabled="false"/>
+            <state-transfer enabled="false"/>
+        </distributed-cache>
+        <distributed-cache name="authenticationSessions" owners="2" statistics="true">
+            <expiration lifespan="-1"/>
+            <persistence passivation="false">
+                <remote-store xmlns="urn:infinispan:config:store:remote:14.0"
+                              cache="authenticationSessions"
+                              raw-values="true"
+                              shared="true"
+                              segmented="false">
+                    <remote-server host="${env.KC_REMOTE_STORE_HOST}"
+                                   port="${env.KC_REMOTE_STORE_PORT}"/>
+                    <connection-pool max-active="16"
+                                     exhausted-action="CREATE_NEW"/>
+                    <security>
+                        <authentication server-name="infinispan">
+                            <digest username="${env.KC_REMOTE_STORE_USERNAME}"
+                                    password="${env.KC_REMOTE_STORE_PASSWORD}"
+                                    realm="default"/>
+                        </authentication>
+                        <encryption protocol="TLSv1.3"
+                                    sni-hostname="${env.KC_REMOTE_STORE_HOST}">
+                            <truststore filename="/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+                                        type="pem"/>
+                        </encryption>
+                    </security>
+                </remote-store>
+            </persistence>
+            <state-transfer enabled="false"/>
+            <state-transfer enabled="false"/>
```

Comment:
`<state-transfer enabled="false">` tag is written twice.

Reply:
Thank you - updated the PR.